OracleDatabase/README.md

```diff
@@ -166,7 +166,8 @@ After the database is setup and/or started the scripts in those folders will be
 SQL scripts will be executed as sysdba, shell scripts will be executed as the current user. To ensure proper order it is
 recommended to prefix your scripts with a number. For example `01_users.sql`, `02_permissions.sql`, etc.
 
-**Note:** The startup scripts will also be executed after the first time database setup is complete.
+**Note:** The startup scripts will also be executed after the first time database setup is complete.  
+**Note:** Use `/u01/app/oracle/scripts/` instead of `/opt/oracle/scripts/setup` for Express Edition.  
```

Comment:
`setup` shouldn't be mentioned as there is also a `startup` folder.
The note should be: > **Note:** Use `/u01/app/oracle/scripts/` instead of `/opt/oracle/scripts/` for Express Edition.

Reply:
Thanks for reviewing, I modified.
Should I fixup my commits?

Reply:
No need. Once @gvenzl approves, I'll squash the commits before I merge.